build/e2e/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM golang:1.18 as build
```

Comment:
Looks like a standard docker build file - what changes do we need for e2e tests?